dockerfiles/pbench/controller/run.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+. /etc/profile.d/pbench-agent.sh
+mkdir -p /var/lib/pbench-agent/tools-default
+cp /root/.ssh/id_rsa /opt/pbench-agent/id_rsa
+sed -i "/^pbench_results_redirector/c pbench_results_redirector = ${pbench_server}" /opt/pbench-agent/config/pbench-agent.cfg
+sed -i "/^pbench_web_server/c pbench_web_server = ${pbench_server}"  /opt/pbench-agent/config/pbench-agent.cfg
+pbench-clear-tools
+ansible-playbook -i /root/inventory /root/pbench/contrib/ansible/openshift/pbench_register.yml
+if [[ "$?" == 0 ]]; then
```

Comment:
Could you add some new lines in between a bit, it's a bit easier to read vs one chunk.